diff --git a/src/plugins/_core/supportHelper.tsx b/src/plugins/_core/supportHelper.tsx
index b88643a3..59ecf4ad 100644
--- a/src/plugins/_core/supportHelper.tsx
+++ b/src/plugins/_core/supportHelper.tsx
@@ -236,7 +236,7 @@ export default definePlugin({
             const roles = GuildMemberStore.getSelfMember(VENCORD_GUILD_ID)?.roles;
             if (!roles || TrustedRolesIds.some(id => roles.includes(id))) return;
 
-            if (!IS_WEB && IS_UPDATER_DISABLED) {
+            if (!IS_WEB && IS_UPDATER_DISABLED && false) {
                 openModal(props => (
                     <ConfirmModal
                         {...props}
