--- a/base/core/core.zsh
+++ b/base/core/core.zsh
@@ -153,7 +153,7 @@ __zplug::core::core::prepare()
 
     # Run compinit if zplug comp file hasn't load
     if (( ! $+functions[_zplug] )); then
-        compinit -C -d "$ZPLUG_HOME/zcompdump"
+        compinit -Cu -d "$ZPLUG_HOME/zcompdump"
     fi
 }
 
--- a/base/core/load.zsh
+++ b/base/core/load.zsh
@@ -31,7 +31,7 @@ __zplug::core::load::from_cache()
 
         # Plugins with defer-level set
         source "$_zplug_cache[defer_1_plugin]"
-        compinit -d "$ZPLUG_HOME/zcompdump"
+        compinit -Cu -d "$ZPLUG_HOME/zcompdump"
         if (( $_zplug_boolean_true[(I)$is_verbose] )); then
             __zplug::io::print::f \
                 --zplug "$fg[yellow]Run compinit$reset_color\n"
